opencensus/common/transports/async_.py

```diff
@@ -145,6 +145,7 @@ def start(self):
             # auto-collection.
             execution_context.set_is_exporter(True)
             self._thread.start()
+            execution_context.set_is_exporter(False)
```

Comment:
I think `execution_context.set_is_exporter(True/False)` should actually be placed around `self.exporter.emit(data)` in line 111. We should be wrapping only the export function with the context.

Reply:
EDIT:
Actually you only need to move `execution_context.set_is_exporter(True)` into `_thread_main` for the context to be set for that thread only.